tests/test_env.py

```diff
@@ -0,0 +1,152 @@
+import os
+from typing import Iterator
+
+import pytest
+
+import pixeltable as pxt
+from pixeltable import exceptions as excs
+from pixeltable.catalog import Catalog
+from pixeltable.config import Config
+from pixeltable.env import Env
+
+
+def _reset_env(reinit: bool) -> None:
+    """Reset the environment for testing."""
+    Catalog.clear()
+    # Reload configs
+    Config.init(config_overrides={}, reinit=True)
+    Env._init_env(reinit_db=reinit)
+
+
+@pytest.fixture(scope='function')
+def test_env_setup(init_env: None) -> Iterator[None]:
+    """Restore PIXELTABLE_DB set by conftest"""
+    test_db = os.environ['PIXELTABLE_DB']
```

Comment:
It'd be safer to use `config_overrides` for this rather than setting and carefully resetting the environment variable - e.g., just take a `db: str` parameter in `_reset_env()`, and pass it along to `config_overrides`.

Right now `PIXELTABLE_DB` is only supported from the environment but I believe it'd be a 1-line change to route it through Config, so that `config_overrides` can be used.